libs/blocks/quiz/quiz.js

```diff
@@ -145,8 +162,13 @@ const App = () => {
                       onOptionClick=${onOptionClick}
                       getOptionsIcons=${getOptionsIcons}
                       handleOnNextClick=${handleOnNextClick} />
+                      <${StepIndicator} 
```

Comment:
I might not have understood the reason, but can we not include the `StepIndicator` component once and show them either in the header or the footer with the breakpoint changes through CSS?

Reply:
Resolving since we talked about this the other day.